run-tests.sh

```diff
@@ -82,7 +82,7 @@ wait_for_sonarqube() {
 
 create_scanner_cache() {
```

Comment:
why do we still need the `run-tests.sh` file? We just migrate to bats tests, let's remove the old test.

Reply:
Benchmarking.
I wanted to ensure I was getting the same results as before. I'll remove it.